docs/src/snap/explanation/ingress.md

```diff
@@ -0,0 +1,79 @@
+# Ingress
+
+In Kubernetes, understanding how inbound traffic is managed inside of your
+cluster can be complex.
+While there is abundant documentation, this explanation provides you with the essentials
+to successfully manage your Canonical Kubernetes cluster.
+
+## Kubernetes Pods and Services
+
+In Kubernetes, the smallest unit is a pod, which encapsulates application containers.
+Since pods are ephemeral and their IP addresses change when destroyed and restarted,
+they are exposed through services.
+Services offer a stable network interface by providing discoverable names and
+load balancing functionality for managing a set of pods.
+For further details on Kubernetes Services,
+refer to the upstream documentation: [Kubernetes Service][Service].
+
+## Ingress
+
+[Ingress][Ingress K8s] is a Kubernetes Resource that manages
+external access by handling both HTTP and HTTPS traffic to services within your cluster.
+Traffic routed through the Ingress is directed to a service,
+which in turn forwards it to the relevant pod
+running the desired application within a container.
+
+The Ingress Resource lets you define rules on how traffic should get handled.
+Learn more about these rules and how to implement them here:
+[Ingress Rules][Ingress Rules].
+
+While the Ingress Resource manages the routing rules for the incoming traffic,
+the [Ingress Controller][Ingress Controller] is responsible for implementing
+those rules by configuring the underlying networking infrastructure of the cluster.
+Ingress does not work without an Ingress Controller.
+
+The Ingress Controller also serves as a layer 7 (HTTP/HTTPS) load balancer
+that routes traffic from outside of your cluster to services inside of your cluster.
+Please do not confuse this with the Kubernetes Service LoadBalancer type.
+
+```{kroki} ../../assets/ingress.puml
+```
+
+With Canonical Kubernetes, enabling default Ingress is easy:
+See [default Ingress][Ingress].
+At the moment of writing, enabling default Ingress will create a
```

Comment:
This sounds a bit like "at the moment we use Cilium but that might change at any moment". Maybe just remove the specific type altogether.